src/neptune/metadata_containers/safe_container.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = ["safety_decorator"]
+
+import functools
+import inspect
+
+from neptune.common.warnings import warn_once
+from neptune.internal.utils.logger import logger
+
+
+def _safe_function(func):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except Exception:
+            warn_once(f"Exception in method {func}")
+            logger.debug("In safe mode exception is ignored", exc_info=True)
```

Comment:
Maybe add any info about an exception.

Reply:
How about adding at least exception message to `warn_once`?

Reply:
How about this?
```
            warn_once(f"Exception in method {func}: {ex.__class__.__name__}")
```

Reply:
Looks ok: 
```
/scratches/scratch_42.py:17: NeptuneDeprecationWarning: Exception in method <function MetadataContainer.set_attribute at 0x119616b90>: MetadataInconsistency
  run["root0/level1"] = "adas"
```